modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-cluster-dpdk_{context}"]
+= Configuring a cluster for DPDK workloads
+
+Use this procedure to configure an {product-title} cluster to run Data Plane Development Kit (DPDK) workloads.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You have installed the Node Tuning Operator.
+* The compute nodes have pre-allocated huge pages and Network Interface Cards (NICs) compatible with SR-IOV.
+
+.Procedure
+. Create a `SriovNetworkNodePolicy` object with the `spec.deviceType` field set to `vfio-pci`.
```

Comment:
Small nit. I think this should be "an", not "a", because AFAIK "SR-IOV" is an initialism (pronounced "ess-ar-eye-oh-vee" etc), not an acronym.